### PR TITLE
Add additional metadata about the fetched token.

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**
Actually dump some of the token data when we make a request, to aid in testing.

### CODE REVIEW

File: testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java [hunk 1]
```diff
@@ -43,9 +43,16 @@
 public class ResultFragment extends Fragment {
 
     public static final String CORRELATION_ID = "correlation_id";
+    public static final String ACCOUNT_TENANT_ID = "account_tenant_id" ;
     static final String ACCESS_TOKEN = "access_token";
     static final String DISPLAYABLE = "displayable";
     static final String STRING_DATA_TO_DISPLAY = "string_data_to_display";
+    public static final String TENANT = "tenant";
```
Comment (shoatman) [inline testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java:50]: Is this home tenant vs account tenant which is from a specific authority which may or may not be the same as the home tenant?

File: testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java [hunk 2]
```diff
@@ -43,9 +43,16 @@
 public class ResultFragment extends Fragment {
 
     public static final String CORRELATION_ID = "correlation_id";
+    public static final String ACCOUNT_TENANT_ID = "account_tenant_id" ;
```
Comment (shoatman) [inline testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java:46]: We probably should distinguish between home tenant and guest tenants and their respective identifiers.  I think the account object has these different IDs are part of tenant profiles.

**Review by shoatman — CHANGES_REQUESTED** [review]
Great to have more information in the result output.  I think we should make clear the difference between home tenant vs other tenants.  Thanks

File: testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java (hunk 1 above)
Comment (AdamBJohnsonx) [reply testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java:50]: To the best of my knowledge here, this is the tenant that issued the token, which may not be the same as the one specified in the authority parameter of the request.  Examining this value could show you the tenant mismatch that was seen in our recent problems.

File: testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java [hunk 3]
```diff
@@ -193,7 +194,25 @@ public boolean onNavigationItemSelected(final MenuItem item) {
...
+                final Map<String, ?> claims1 = mAuthResult.getAccount().getClaims();
+                if (claims1 != null) {
+                    for (Map.Entry<String, ?> e : claims1.entrySet()) {
+                        claims.append(e.getKey() + " -> " + String.valueOf(e.getValue()));
+                    }
+                } else {
+                    claims.append("NONE!");
+                }
+                bundle.putString(ResultFragment.CLAIMS, claims1.toString());
+                bundle.putString(ResultFragment.ACCOUNT_TENANT, mAuthResult.getAccount().getTenantId());
```
Comment (rpdome) [inline testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java:210]: nit: ACCOUNT_TENANT_ID?

Comment (rpdome) [reply testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java:210]: alternatively, we could just print the whole mAuthResult.

**Pull request closed without merging**